src/main/kotlin/com/grepp/nbe1_3_team04/reservation/domain/ReservationStatus.kt

```diff
@@ -0,0 +1,10 @@
+package com.grepp.nbe1_3_team04.reservation.domain
+
+
+enum class ReservationStatus (private val description: String) {
```

Comment:
여기도 별건 아닌데
```suggestion
enum class ReservationStatus (
private val description: String
) {
```

이게 더 보기 좋을거같아여 제 코드도 제대로 되어있는지는 모르겠지만.. 포매팅을 자동으로 맞출수있는 방법이 있으면 좋겠네요

Reply:
넵 감사합니다!